src/utils/queryString/sortString.ts

```diff
@@ -36,6 +36,23 @@ export const toSortString = (
     : undefined;
 };
 
+// takes an array of sort orders and translates it into the sorts query param
+// the function above is similar, but it's tied to the antd table
+export const updateSortString = (sorts: SortOrder[]) => {
+  let sortStrings: string[] = [];
+
+  sorts.forEach((sort) => {
+    if (sort.Direction !== undefined) {
+      const singleSortString = `${sort.Key}:${sort.Direction}`;
+      sortStrings = sortStrings.concat(singleSortString);
+    }
+  });
+
+  return sortStrings.some((s) => s)
+    ? sortStrings.filter(Boolean).join(";")
+    : undefined;
+};
```

Comment:
I discovered recently that queryString can do this natively. It  can generate a query param like this natively. https://www.npmjs.com/package/query-string

Reply:
I gave this a try but I'm not sure if it'll work as intended...

I read the documentation and I think what needs to happen is that the `SortOrder[]` needs to be combined into an object that looks like `{ sorts: ["STATUS:ASC", "DURATION:ASC", "NAME:DESC"] }` and then use the `stringify() `function with  a separator of `;`.

This function will return `"sorts=STATUS:ASC;DURATION:ASC;NAME:DESC"`, but because `updateQueryParams` updates a specific field the param will end up looking like `"sorts=sorts=STATUS:ASC;DURATION:ASC;NAME:DESC"` which is not correct. I might just be not understanding something though 🥲